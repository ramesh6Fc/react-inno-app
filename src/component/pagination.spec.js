import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Pagination from './pagination';


describe('The Pagination page', () => {
    it ('Pagination page renderer as expected',() => {
        const tree = renderer
                        .create(<Pagination />)
                        .toJSON();
                        console.log(tree.children, "Pagination page elements");

                        expect(tree).toMatchSnapshot ();
    })

})