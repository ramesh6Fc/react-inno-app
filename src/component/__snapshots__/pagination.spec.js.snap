// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Pagination page Pagination page renderer as expected 1`] = `
<div>
  <nav>
    <ul
      className="pagination float-right"
    >
      <li
        className="page-item"
      >
        <select
          onChange={[Function]}
          style={
            Object {
              "margin": "5px",
            }
          }
        >
          <option>
            5
          </option>
          <option>
            10
          </option>
          <option>
            20
          </option>
          <option>
            50
          </option>
        </select>
        <span
          style={
            Object {
              "marginRight": "10px",
            }
          }
        >
          /Page:  
        </span>
      </li>
      <li>
        <a
          aria-label="Previous"
          href="#/"
        >
          <span
            aria-hidden="true"
            onClick={[Function]}
            style={
              Object {
                "fontSize": "large",
                "marginRight": "10px",
              }
            }
          >
            «
          </span>
        </a>
      </li>
      <li
        className="page-item"
      >
        <select
          onChange={[Function]}
          style={
            Object {
              "margin": "5px",
            }
          }
        />
      </li>
      <li
        lassName="page-item"
      >
        <a
          aria-label="Next"
          href="#/"
        >
          <span
            aria-hidden="true"
            onClick={[Function]}
            style={
              Object {
                "fontSize": "large",
                "marginLeft": "10px",
              }
            }
          >
            »
          </span>
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
